packages/documentation/docs/pages/FileSystem.md

```diff
@@ -2,7 +2,13 @@
 
 The [UI5 FS](https://github.com/SAP/ui5-fs) provides a UI5-specific file system abstraction.
 
-[**API Reference**](https://ui5.github.io/cli/v4/api/){: .md-button .sap-icon-initiative }
+<script setup>
+import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vue"
+</script>
+
+<div style="margin: 1rem 0;">
+  <VPButton class="no-decoration" text="📚 API Reference" href="https://ui5.github.io/cli/v4/api/"/>
```

Comment:
v4?

I wonder why we can't use relative links here.

Reply:
It' should be v5, indeed and I have adjusted it, but It used to be that way so far.
I was trying to just adjust the formatting and eventually something broken/wrong.